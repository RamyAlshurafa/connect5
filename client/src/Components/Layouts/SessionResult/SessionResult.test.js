import React from "react";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router-dom";

import SessionResults from "./index";

test("snapshot for 'SessionResults' layout", () => {
  const context = {};
  const match = {
    params: {
      sessionId: "sessionId",
      sessionType: "sessionType",
    },
  };

  const component = renderer.create(
    <StaticRouter context={context}>
      <SessionResults match={match} />
    </StaticRouter>,
  );
  expect(component).toMatchSnapshot();
});
