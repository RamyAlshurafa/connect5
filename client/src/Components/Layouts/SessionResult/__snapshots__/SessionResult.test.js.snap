// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for 'OverallResults' layout 1`] = `
<div
  className="sc-bdVaJa fIAUCv"
>
  <div
    className="sc-gZMcBi VIFGE"
  >
    <h1
      className="sc-bwzfXH dZCpGN"
    >
      Session Details
    </h1>
  </div>
  <div
    className="sc-htpNat rqoCe"
  >
    <button
      className="sc-bxivhb hFsRiX"
      name="overall"
      onClick={[Function]}
    >
      Overall Results
    </button>
    <button
      className="sc-bxivhb kVBIuy"
      name="individual"
      onClick={[Function]}
    >
      Individual Responses
    </button>
  </div>
  <div>
    <div
      className="sc-ifAKCX gVwUCl"
    >
      <p
        className="sc-EHOje hKuwFc"
      >
        <span
          className="sc-bZQynM kYemYx"
        >
          Number of Replies:
        </span>
         
        (out of
         
         
        )
      </p>
      <p
        className="sc-EHOje hKuwFc"
      >
        <span
          className="sc-bZQynM kYemYx"
        >
          Completion Rate:
        </span>
        NaN
        %
      </p>
    </div>
    <h3
      className="sc-gzVnrw gdyVTx"
    >
      Responses
    </h3>
  </div>
</div>
`;

exports[`snapshot for 'SessionResults' layout 1`] = `
<div
  className="sc-bdVaJa fIAUCv"
>
  <div
    className="sc-gZMcBi VIFGE"
  >
    <h1
      className="sc-bwzfXH dZCpGN"
    >
      Session Details
    </h1>
  </div>
  <div
    className="sc-htpNat rqoCe"
  >
    <button
      className="sc-bxivhb hFsRiX"
      name="overall"
      onClick={[Function]}
    >
      Overall Results
    </button>
    <button
      className="sc-bxivhb kVBIuy"
      name="individual"
      onClick={[Function]}
    >
      Individual Responses
    </button>
  </div>
  <div>
    <div
      className="sc-ifAKCX gVwUCl"
    >
      <p
        className="sc-EHOje hKuwFc"
      >
        <span
          className="sc-bZQynM kYemYx"
        >
          Number of Replies:
        </span>
         
        (out of
         
         
        )
      </p>
      <p
        className="sc-EHOje hKuwFc"
      >
        <span
          className="sc-bZQynM kYemYx"
        >
          Completion Rate:
        </span>
        NaN
        %
      </p>
    </div>
    <h3
      className="sc-gzVnrw gdyVTx"
    >
      Responses
    </h3>
  </div>
</div>
`;
